dotnet/src/Azure.Iot.Operations.Services/AssetAndDeviceRegistry/Models/ProtocolConverter.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Iot.Operations.Services.AssetAndDeviceRegistry.AdrBaseService;
+using Azure.Iot.Operations.Services.AssetAndDeviceRegistry.AepTypeService;
+
+namespace Azure.Iot.Operations.Services.AssetAndDeviceRegistry.Models;
+
+public static class ProtocolConverter
```

Comment:
This is a class for converting from public APIs to code gen, right? If so, this class shouldn't be public since no user should need to access it